CollateralAccount/CollateralAccount.md

```diff
@@ -0,0 +1,106 @@
+---
+CIP: ?
+Title: Collateral Account for HD Wallets
```

Comment:
How about calling this a `Collateral Address`?

Reply:
Yes, that's better. "Collateral addresses for HD wallets".